diff --git a/dist/layout.js b/dist/layout.js
index 57a511faacf75820f822be305d66c61494ca998a..34968f1b8027a9a4a1b4f7478b545f9cb6b3da5b 100644
--- a/dist/layout.js
+++ b/dist/layout.js
@@ -23,19 +23,20 @@ const Layout = (t0) => {
     themeConfig = $[2];
   }
   let data;
-  if ($[3] !== themeConfig) {
+  {
     const {
       data: t12,
       error
-    } = LayoutPropsSchema.safeParse(themeConfig);
+    } = LayoutPropsSchema.safeParse({
+      ...themeConfig,
+      children
+    });
     data = t12;
     if (error) {
       throw z.prettifyError(error);
     }
     $[3] = themeConfig;
     $[4] = data;
-  } else {
-    data = $[4];
   }
   let banner;
   let footer;
